README.md

```diff
@@ -1,2 +1,12 @@
 # stardog-userrole-operator
 A Kubernetes operator to manage Stardog users and roles.
+
+# Generating the REST client
+
+The package stardogrest is a REST client generated by [autorest](http://azure.github.io/autorest/) based on the [stardogrest/stardog_swagger.yaml](stardogrest/stardog_swagger.yaml) file. If the stardog REST API changes, the [stardogrest/stardog_swagger.yaml](stardogrest/stardog_swagger.yaml) should be updated to reflect the changes, and then autorest should be run again with the following command:
+
+```
+autorest --go --package-name=github.com/vshn/stardog-userrole-operator/stardogrest --namespace=stardogrest --input-file=stardogrest/stardog_swagger.yaml --output-folder=stardogrest
+```
+
+The auto-generation unfortunately fails the linting step, so after regenerating the code, be sure to run `make lint` on it before committing. 
```

Comment:
If the make target has formatting in it, then this isn't necessary anymore

Reply:
Exactly. I have now deleted this line.